lib/tasks/port_milestones_to_timelines.rake

```diff
@@ -0,0 +1,22 @@
+namespace :port_milestones_to_timelines do
```

Comment:
we already have a `milestones.rake` file... can we add this task to that namespace?
also, we need to clean up our `full_import task` in `dm.rake`
line 19 calls `Rake::Task['milestones:milestones_transfer'].execute` which we don't need to call anymore. but let's keep the task just in case...
We can remove these from the README as well

Reply:
check out change c900a25' in the dm-2018 branch I believe I have it all covered.